src/Pages/ItemList/Components/ItemContainer/GridPanel.js

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import { SORT_KEY } from "../../itemData";
+
+function GridPanel({ products, sort, getSortText }) {
+  return (
+    <GridContainer>
+      {products.map(({ title, releaseDate, market, media: { imageUrl } }) => (
+        <ProductContainer key={title}>
+          <ProductWrapper>
+            <ProductLink to="#">
+              <ProductImageWrapper>
+                <ProductImage
+                  alt={title}
+                  src={imageUrl}
+                  onError={(e) => {
+                    e.target.onerror = null;
+                    e.target.src = "/Images/empty_item.jpg";
+                  }}
+                />
+              </ProductImageWrapper>
+              <ProductDescWrapper>
+                <ProductTitle>
+                  {title.length < 45
+                    ? title
+                    : title.substring(0, title.indexOf("("))}
+                </ProductTitle>
+                <div>
+                  <ProductPriceName>
+                    {sort === "highest_bid" ? "highest bid" : "lowest ask"}
+                  </ProductPriceName>
+                  <ProductPrice>{"$" + market.lowestAsk}</ProductPrice>
+                </div>
+                {sort === "lowest_ask" ? (
+                  sort === "highest_bid" ? (
+                    <></>
+                  ) : (
+                    <></>
+                  )
+                ) : (
+                  <ProductSortText>
+                    {getSortText(
+                      "grid",
+                      sort,
+                      sort === "release_date"
+                        ? releaseDate
+                        : market[SORT_KEY[sort]]
+                    )}
```

Comment:
```suggestion
                {sort !== "lowest_ask" && <ProductSortText>
                    {getSortText(
                      "grid",
                      sort,
                      sort === "release_date"
                        ? releaseDate
                        : market[SORT_KEY[sort]]
                    )}
```
그냥 이렇게 하면 안되는건가요? 확인 해보시고 다음 PR 때 반영해주세요.